textattack/shared/checkpoint.py

```diff
@@ -0,0 +1,128 @@
+import os
+import pickle
+import datetime
+from textattack.shared import utils
+from textattack.attack_results import SuccessfulAttackResult, FailedAttackResult, SkippedAttackResult
+
+class CheckPoint:
+    """ An object that stores necessary information for saving and loading checkpoints
+    
+        Args:
+            time (float): epoch time representing when checkpoint was made
+            args: command line arguments of the original attack
+            log_manager (AttackLogManager)
+    """
+    def __init__(self, time, args, log_manager):
```

Comment:
maybe `time` should be optional and should default to `time.time()`?

Reply:
Good point